ui/src/app/components/runtime-configuration-panel/customize-panel-footer.spec.tsx

```diff
@@ -28,7 +28,6 @@ const gcePersistentDisk = stubDisk();
 const analysisConfig = toAnalysisConfig(currentRuntime, gcePersistentDisk);
 const existingAnalysisConfig = analysisConfig;
 const defaultProps: CustomizePanelFooterProps = {
-  disableControls: false,
```

Comment:
this was overly restrictive - checking a different set of states besides `canDeleteRuntime()`